pyproject.toml

```diff
@@ -1,7 +1,36 @@
-[tool.isort]
-profile = "black"
-lines_after_imports = 2
-src_paths = ["kinto-remote-settings", "tests"]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".git",
+    "__pycache__",
+    ".venv",
+    "node_modules",
+]
+
+[tool.ruff.lint]
+preview = true
+select = [
+    # pycodestyle
+    "E", "W",
+    # flake8
+    "F",
+    # isort
+    "I"
```

Comment:
It seems that ruff comes with its own import sorter. Can we get away with not specifying `isort` linting rules and letting the formatter handle it all? 

I know we set `lines-after-imports = 2` below, but if we were fine without enabling that rule, it would be mean fewer lines of config overall.

Reply:
> It seems that ruff comes with its own import sorter. 

Yes, it's enabled with `I` rules!

> if we were fine without enabling that rule, it would be mean fewer lines of config overall.

I'm +1 on keeping this config line, since it's the standard we use on our other repos